### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by ikoevska**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+===
+
+### New
+
+* [Implemented #305](https://github.com/NativeScript/nativescript-cli/issues/305), [#322](https://github.com/NativeScript/nativescript-cli/issues/322): You can quickly add or update your platform runtime to a specific version by running `tns platform update platform@version`<br/>For example: `tns platform update ios@0.9.2-beta`
+* [Implemented #302](https://github.com/NativeScript/nativescript-cli/issues/302): You can configure proxy settings for the NativeScript CLI.
+
+### Fixed
+
+* [Fixed #312](https://github.com/NativeScript/nativescript-cli/issues/312): `tns platform add ios` does not preserve your app ID, if not default.
```
Comment (rosen-vladimirov) [inline CHANGELOG.md]: ping @Fatme  - have we fixed this one for 0.9.2 or we've postponed it for 0.10 ?

Comment (Fatme) [inline CHANGELOG.md]: We've posponed it for `0.10`.

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by dtopuzov** [comment]
:+1:

**Comment by Fatme** [comment]
:+1:

**Pull request merged**